Include/objimpl.h

```diff
@@ -111,6 +111,9 @@ PyAPI_FUNC(void) PyObject_Free(void *ptr);
 #define PyObject_Del            PyObject_Free
 #define PyObject_DEL            PyObject_Free
 
+/* for source compatibility with 2.2 */
+#define _PyObject_Del           PyObject_Free
```

Comment:
Why do we need this? It doesn't look like it's needed in pygrate3?

Reply:
Its used somewhere in the cmp port code

Reply:
Actually same as the line 112, I called the already defined one.

Reply:
I guess my question is: we don't see to use `_PyObject_Del` anywhere otherwise the PR wouldn't have compiled before this addition?

Reply:
Ah I see you've got rid of it in https://github.com/softdevteam/pygrate3/pull/4/commits/90060cb3650c395079d686563bd11a07437cc75f.